charts/influxdb-enterprise/templates/data-statefulset.yaml

```
@@ -88,6 +88,10 @@ spec:
           env:
           - name: RELEASE_NAME
             value: {{ include "influxdb-enterprise.fullname" . }}
+          {{- range $key, $val := .Values.data.env }}
+          - name: {{ $key }}
+            value: {{ $val | quote }}
+          {{- end}}
```

Other charts use simpler way, which also does not unnecessarily quote values
```suggestion
          {{- if .Values.data.env }}
{{ toYaml .Values.data.env | indent 10 }}
          {{- end}}
```